### PR TITLE
Add deployment smoke-test scripts

### PR DESCRIPTION
**Pull request by higs4281**
These scripts are intended to make sure the basics are working
immediately after a deployment. The scripts make requests that bypass
Akamai to check that pages and static assets are returning `200`
status codes.

This refactors the http_smoke_test script and moves it out of our
Jenkins-as-code repo. After merging, we can adjust the JAC job 
to get the script here.

This PR also updates our testing docs to describe using `tox -e fast`
and how to see the coverage report.

## Additions
- http_smoke_test.py
- static_asset_smoke_test.py
- related tests
- `show_reports.sh` to check the coverage report from the site root

## Testing

- `tox -e fast` will test the scripts
- to see the scripts in action, run them from the scripts directory:
```
cd cfgov/scripts/
python http_smoke_test.py -v --full
python static_asset_smoke_test.py -v askcfpb retirement
```

The static_asset script will check the home page by default, plus any sub-urls you want to add.
In Jenkins, these tests will be run to bypass cache.

## Review

- @chosak @richaagarwal @willbarton @grapesmoker @contolini 

## Todos

- Reconfigure the static_assets script to check any one static asset only once.

## Checklist

* [x] Project documentation and CHANGELOG has been updated

### CODE REVIEW

**Comment by richaagarwal** [comment]
Could these live in `v1/tests/` instead of under `scripts` ?

File: cfgov/scripts/http_smoke_test.py [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    count = 0
+    timeouts = []
+    failures = []
+    starter = time.time()
+    if full:
+        url_list = FULL_RUN
+    else:
+        url_list = SHORT_RUN
+    for url_suffix in url_list:
+        # print(url_suffix)
```
Comment (richaagarwal) [inline cfgov/scripts/http_smoke_test.py]: Can this comment be removed, or updated?  I could see it being helpful to log each URL, so maybe something like `logger.info(url_suffix)` instead?

Comment (higs4281) [inline cfgov/scripts/http_smoke_test.py]: @richaagarwal done.

**Comment by higs4281** [comment]
> Could these live in v1/tests/ instead of under scripts ?

Two thoughts: 
- The scripts are not code tests themselves; in fact they have their own tests.
- The scripts are not v1-specific. They're site-wide, and intended for external (Jenkins) use.

File: docs/testing.md [hunk 2]
```diff
@@ -129,11 +129,24 @@ The audit will run against
...
+
+If you haven't changed any installed packages and you don't need to test all migrations, you can run a much faster Python code test using:
+```
+tox -e fast
+```
+
+To see Python code coverage information, run
+```
+./show_coverage.sh
+``
```
Comment (willbarton) [inline docs/testing.md]: This whole expanded section of docs.

good-job

**Comment by chosak** [comment]
:+1:

Minor request: could you run `flake8` against these files? I notice some minor things like extra whitespace, etc.

**Comment by richaagarwal** [comment]
@higs4281 Did you mean to close this without merging?

**Pull request merged**